nix/pkgs/haskell/haskell.nix

```diff
@@ -26,6 +27,9 @@ let
         name = "plutus-apps";
       };
     sha256map = import ./sha256map.nix;
```

Comment:
Do we need this or can we do `sha256Map = {}`? (<-- we do this in `plutus`)

Reply:
`plutus` managed to get rid of all `source-repository-package`s, so they don't need `sha256Map` anymore. That said I should remove from `nix/pkgs/haskell/sha256map.nix` the hashes that are not needed anymore. Good catch.